tests/route/test_route_perf.py

```diff
@@ -22,7 +22,46 @@
 
 ROUTE_TABLE_NAME = "ASIC_STATE:SAI_OBJECT_TYPE_ROUTE_ENTRY"
 DEFAULT_NUM_ROUTES = 10000
-DEAFULT_M0_MX_NUM_ROUTES = 500
+
+route_scale_per_role = {
+    "m0": {
+        "ipv4": 500,
+        "ipv6": 500
+    },
+    "mx": {
+        "ipv4": 500,
+        "ipv6": 500
+    },
+    "t0": {
+        "ipv4": 40000,
```

Comment:
Why 40k ipv4 and 8k ipv6? Any spec requirement or just use a relatively big number?

Reply:
Per @prsunny, this is the committed number for these topologies and more than double the typical production scale. In the production scenario, the issue of tcam table full was hit at 8k ipv6 route scale.